content/browser/mojom/BUILD.gn

```diff
@@ -0,0 +1,5 @@
+import("//mojo/public/tools/bindings/mojom.gni")
```

Comment:
we shouldn't be using `content` as a prefix for anything, this should go in the cosmetic filters component

Reply:
not an issue anymore